src/server/components/lists/searches/covid-test-provider.ts

```diff
@@ -25,25 +26,28 @@ export async function searchCovidTestProvider(
   req: Request,
   res: Response
 ): Promise<void> {
-  let params = getAllRequestParams(req);
-  const { serviceType, country, region, resultsTurnaround } = params;
-  const countryName = formatCountryParam(country as string);
-  params = { ...params, country: countryName as CountryName };
-  const searchResults =
-    await CovidTestSupplierListItem.findPublishedCovidTestSupplierPerCountry({
+  try {
+    let params = getAllRequestParams(req);
+    const { serviceType, country, region, resultsTurnaround } = params;
+    const countryName = formatCountryParam(country as string);
+    params = { ...params, country: countryName as CountryName };
+    const searchResults = await CovidTestSupplierListItem.findPublishedCovidTestSupplierPerCountry({
```

Comment:
this should be handled with the prisma upgrade pr, however, would be better to wrap this and return an empty searchResults to the results-page